controllers/user.js

```diff
@@ -75,7 +80,7 @@ export default class UserController {
       body: { rememberMe, password },
       user
     } = req;
-    const passwordMatch = compareSync(password, user.password);
+    const passwordMatch = bcrypt.compareSync(password, user.password);
```

Comment:
@youngestdj are you sure this change works?

Reply:
Yes it does. I have manually tested it and tests that utilize that line pass